app/src/examples/RuntimeTests/tests/getRelativeCoords/relativeCoords.test.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import Animated, {
+  runOnUI,
+  measure,
+  getRelativeCoords,
+  ComponentCoords,
+  useAnimatedRef,
+  useSharedValue,
+} from 'react-native-reanimated';
+import {
+  describe,
+  test,
+  expect,
+  expectNotNullable,
+  render,
+  wait,
+  registerValue,
+  getRegisteredValue,
+} from '../../ReanimatedRuntimeTestsRunner/RuntimeTestsApi';
+
+const REGISTERED_VALUE_KEY = 'sv';
+
+const CoordsComponent = ({ testStyle }: { testStyle: object }) => {
+  const coordsSv = useSharedValue<ComponentCoords | null>(null);
+  registerValue(REGISTERED_VALUE_KEY, coordsSv);
+  const bRef = useAnimatedRef();
+  const sRef = useAnimatedRef();
+
+  const onLayoutMeasure = () => {
+    runOnUI(() => {
+      const measured = measure(sRef);
+      if (measured !== null) {
+        coordsSv.value = getRelativeCoords(bRef, measured.pageX, measured.pageY);
+      }
+    })();
+  };
+
+  return (
+    <Animated.View style={styles.container}>
+      <Animated.View ref={bRef} style={[styles.bigBox, testStyle]}>
+        <Animated.View ref={sRef} style={styles.smallBox} onLayout={onLayoutMeasure} />
+      </Animated.View>
+    </Animated.View>
+  );
+};
+
+describe('getRelativeCoords', () => {
+  test.each(TEST_CASES)('getCoords with style: ${style}', async ({ style, expectedValueX, expectedValueY }) => {
```

Comment:
See comment below:
Once you make TEST_CASES compact you could:
```suggestion
  test.each([
      ['flex-start', 'flex-start', 0, 0],
      ['flex-start', 'center', 50, 0], //... 
  ])('getCoords with style: ${style}', async ({ style, expectedValueX, expectedValueY }) => {
```